docs/solidity-by-example.rst

```diff
@@ -129,6 +129,8 @@ of votes.
                 to = voters[to].delegate;
 
                 // We found a loop in the delegation, not allowed.
+                // Will note catch loops that don't include msg.sender
```

Comment:
It should not be possible to generate such loops, because they would have been detected here earlier.